pandas/core/common.py

```diff
@@ -1888,60 +1890,58 @@ def _possibly_convert_objects(values, convert_dates=True,
                               convert_timedeltas=True):
     """ if we have an object dtype, try to coerce dates and/or numbers """
 
+    # If 1 flag is coerce, ensure 2 others are False
+    conversions = (convert_dates, convert_numeric, convert_timedeltas)
+    if 'coerce' in conversions:
+        coerce_count = sum([c == 'coerce' for c in conversions])
+        if coerce_count > 1:
+            raise ValueError("'coerce' can be used at most once.")
+
+        false_count = sum([not c for c in conversions])
+        if false_count != 2:
+            import warnings
+            warnings.warn("Soft conversion flags ignored when using 'coerce'",
+                          RedundantSettingWarning)
```

Comment:
this whole part (about seeing if we have multiple 'coerces' belongs below where you actually do it. So that you have 1 place where you test if you have more than 1 coerce (ValueError), or at least 1, then branch to the coerce part, then simply return.